client/updateCenter_test.go

```diff
@@ -117,4 +117,33 @@ var _ = Describe("update center test", func() {
 			Expect(status.RestartRequiredForCompletion).Should(BeTrue())
 		})
 	})
+
+	Context("SitePlugin", func() {
```

Comment:
I believe that you want to test method `GetSite`. https://github.com/jenkins-zh/jenkins-cli/pull/132/files#diff-8154d05af853407f220c9381990ef8dbR163

So, please consider using the real method name instead of another one.